### PR TITLE
Initial support for Develco Smart Cable

### PR DESCRIPTION
**Pull request by Smanar**
Second try, sorry
for https://github.com/dresden-elektronik/deconz-rest-plugin/issues/3667

### CODE REVIEW

**Comment by SwoopX** [comment]
@Smanar with the new one, you seem to have missed some spots which were previously considered. Can you double check?

**Comment by Smanar** [comment]
Better now ?

File: bindings.cpp [hunk 1]
```diff
@@ -1700,7 +1700,8 @@ bool DeRestPluginPrivate::sendConfigureReportingRequest(BindingTask &bt)
         rq2.maxInterval = 300;
         if (sensor && (sensor->modelId() == QLatin1String("SmartPlug") || // Heiman
                        sensor->modelId() == QLatin1String("902010/25") || // Bitron
-                       sensor->modelId() == QLatin1String("SKHMP30-I1") ||// GS smart plug
+                       sensor->modelId() == QLatin1String("SKHMP30-I1") || // GS smart plug
+                       sensor->modelId() == QLatin1String("SMRZB-1") || // Develco smart cable
```
Comment (SwoopX) [inline bindings.cpp:1704]: This one is not necessary, since it already reports in W as I recall

**Review by SwoopX — CHANGES_REQUESTED** [review]
The device would need an addition around L1750 in `bindings.cpp` to report 1V steps. Apart from the 2 comments, it should be fine. Thanks!

File: bindings.cpp (hunk 1 above)
Comment (Smanar) [reply bindings.cpp]: Done.

**Review by SwoopX — APPROVED** [review]

**Comment by SwoopX** [comment]
👍 That should do the trick.

**Comment by manup** [comment]
Hi, looks like initial support for this device?
If yes can you please change the PR title and prepent "Initial support for".

**Comment by manup** [comment]
Can you please sync this PR this latest master, then it's ready to merge.

**Pull request merged**